test/framework/github.py

```diff
@@ -240,6 +240,13 @@ def test_validate_github_token(self):
 
         self.assertTrue(gh.validate_github_token(self.github_token, GITHUB_TEST_ACCOUNT))
 
+    def test_find_easybuild_eb(self):
+        """Test for find_easybuild_eb function"""
+        path = gh.find_easybuild_eb()
+        expected = os.path.join('e', 'EasyBuild', 'EasyBuild-[1-9]+\.[1-9]+\.[1-9]+\.eb')
+        regex = re.compile(expected)
+        self.assertTrue(re.search(regex, path))
```

Comment:
also make sure the returned path exists: `self.assertTrue(os.path.exists(path))`